### PR TITLE
Version 1 release :partying_face: 

### PR DESCRIPTION
**Pull request by willGraham01**
Staging the changes for the upcoming release of `v1.0.0` of `brainreg` along with `v1` of `BrainGlobe`.

See https://github.com/brainglobe/BrainGlobe/issues/38 for details.

Upon merging, we will:
- Push & tag `brainreg` `v1.0.0`, which will complete the upload to PyPI
- Deprecate `brainreg-napari` https://github.com/brainglobe/BrainGlobe/issues/37

### CODE REVIEW

**Comment by adamltyson** [comment]
I just thought, is this going to break cellfinder? e.g. https://github.com/brainglobe/cellfinder/blob/21f724b46c84cf2c2489d1d4347c6c19f9277b45/cellfinder/main.py#L45

**Comment by willGraham01** [comment]
> Is this going to break cellfinder? e.g. https://github.com/brainglobe/cellfinder/blob/21f724b46c84cf2c2489d1d4347c6c19f9277b45/cellfinder/main.py#L45

Yes (and the workflows repo dev branch too). Let me open a cellfinder PR to address the imminent breakage and add it to the list of things to update in during the breakage period. We won't be able to validate the fix until we get into the breakage-hour of course, but it should just be a substitution of `brainreg` -> `brainreg.core`.

**Review by alessandrofelder — DISMISSED** [review]
One tiny comment - and yes, we should make sure that we don't break cellfinder if possible :)

File: README.md [hunk 1]
```diff
@@ -9,165 +9,150 @@
...
-
-**N.B. There is also a [napari plugin](https://github.com/brainglobe/brainreg-napari) if you'd rather use brainreg with a graphical user interface. Currently this interface is slightly limited compared to the command line tool**
-
-This software is at a very early stage, and was written with our data in mind.
-Over time we hope to support other data types/formats. If you have any issues, please get in touch [on the forum](https://forum.image.sc/tag/brainglobe) or by
+Documentation for both the command-line tool and graphical interface can be found [here](https://brainglobe.info/documentation/brainreg/index.html).
+If you have any issues, please get in touch [on the forum](https://forum.image.sc/tag/brainglobe), [on Zulip](https://brainglobe.zulipchat.com/), or by
 [raising an issue](https://github.com/brainglobe/brainreg/issues).
 
+For segmentation of bulk structures in 3D space (e.g. injection sites, Neuropixels probes), please see [brainreg-segment](https://github.com/brainglobe/brainreg-segment).
```
[the remark this answers is not in the document]
Comment (willGraham01) [reply README.md:20]:
```suggestion
For segmentation of bulk structures in 3D space (e.g. injection sites, Neuropixels probes), please see [brainglobe-segmentation](https://github.com/brainglobe/brainglobe-segmentation).
```

**Pull request merged**